lib/config.js

```diff
@@ -21,37 +28,35 @@ async function getGitRev () {
 }
 
 async function getAppiumConfig () {
+  let stat = await B.promisify(fs.stat)('./build');
```

Comment:
I think we should probably not assume a particular cwd? Maybe this should be `path.resolve(__dirname, "..")` instead of `'./build'`?